src/objects/bunch/bunch.scss

```diff
@@ -0,0 +1,45 @@
+@use '../../design-tokens/sizes.yml';
+
+/**
+ * Container
+ *
+ * 1. Child elements of varying sizes look a bit off if they aren't centered.
+ * 2. Keeps z-index values of child elements relative to this to avoid
+ *    disrupting outside stacking order.
+ */
+
+.o-bunch {
+  align-items: center; /* 1 */
+  display: flex;
+  position: relative; /* 2 */
+}
+
+/**
+ * Child elements
+ *
+ * 1. Prevent unintended squashing of items in flex container.
+ * 2. Overlap with adjacent items to for a unified visual appearance.
+ * 3. It feels odd to have the first element underneath the last. In lieu of a
+ *    a built-in CSS method of reversing the natural `z-index` order, we use
+ *    a custom property that we'll set with `nth-child` a little further down.
+ *
+ * @see https://github.com/w3c/csswg-drafts/issues/1026#issuecomment-385285148
+ */
+
+.o-bunch > * {
+  flex: none; /* 1 */
+  margin-right: (sizes.$overlap-small * -1); /* 2 */
```

Comment:
Smart! I likely would have ended up doing something like this instead (I like your version better)

```
.o-bunch > * + * {
   margin-left: (sizes.$overlap-small * -1); /* 2 */ 
}
```